src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -398,6 +402,13 @@ public IInternalContest fromYaml(IInternalContest contest, String[] yamlLines, S
             setCcsTestMode(contest, ccsTestMode);
         }
         
+        boolean stopOnFirstFail = fetchBooleanValue(content, STOP_ON_FIRST_FAILED_TEST_CASE_KEY, false);
+        if (stopOnFirstFail) {
+            setStopOnFirstFailedTestCase (contest, stopOnFirstFail);
+        }
+        
+        System.out.println("debug 22 for stopOnFirstFail is " +contest.getContestInformation().isStopOnFirstFailedtestCase() + " "+directoryName );
```

Comment:
As someone keeps pointing out to me, debug print statements should be removed.

Reply:
:)  pushed fix.